src/models/apis/googleMaps.ts

```diff
@@ -451,10 +453,14 @@ export default class GoogleMaps implements IMapFunctions {
     }
 
     public alterCircleOptions(circles: any[], options: CircleAlterOptions) {
-        let newOptions = {};
-
         circles.forEach((circle) => {
-            newOptions = {
+            const latlng = options.center !== null && options.center !== undefined && options.center.length > 0 ?
```

Comment:
Não daria pra simplificar?
```suggestion
            const latlng = options.center && options.center.length > 0 ?
```

Reply:
Alteração adicionada